GearSwapper/src/main/java/net/runelite/client/plugins/gearswapper/GearSwapper.java

```diff
@@ -322,6 +322,31 @@ private void executeSwap(List<WidgetItem> equip, List<Widget> remove)
 		return equipped;
 	}
 
+	private void spec()
+	{
+		if (client.getWidget(WidgetInfo.MINIMAP_SPEC_SPRITE).getSpriteId() == 1607 || client.getWidget(WidgetInfo.MINIMAP_SPEC_SPRITE).getSpriteId() == 1608)
+		{
+			handleSwitch(client.getWidget(WidgetInfo.MINIMAP_SPEC_ORB).getBounds());
+		}
+		else
+		{
+			flexo.keyPress(tabUtils.getTabHotkey(Tab.ATTACK));
+			try
```

Comment:
since we're already injecting flexo, just use flexos delay, I'll update mine to do this as-well. (just looks nicer)